hysds/user_rules_job.py

```diff
@@ -79,14 +79,14 @@ def update_query(job_id, rule):
     return final_query
 
 
-def evaluate_user_rules_job(job_id, alias=STATUS_ALIAS):
+def evaluate_user_rules_job(job_id, index=None):
```

Comment:
Should we default to JOB_STATUS_ALIAS?

```suggestion
def evaluate_user_rules_job(job_id, index=JOB_STATUS_ALIAS):
```

Reply:
we default to `JOB_STATUS_ALIAS` when evaluating the query here:
```python
result = mozart_es.es.search(index=index or JOB_STATUS_ALIAS, body=final_qs)
```

Reply:
@mcayanan: does that work for you?

Reply:
oh my bad. I didn't see that. No change needed then. I'll resolve.